### PR TITLE
Fixes #519 - Unable to run Eclipse Transformer on Java 21 using the Maven plugin

### LINKED ISSUE #519: Unable to run Eclipse Transformer on Java 21 using the Maven plugin
To make it work I had to add the following dependency to the 0.5.0 version of the Maven plugin:

```
<dependency>
  <groupId>biz.aQute.bnd</groupId>
  <artifactId>biz.aQute.bnd</artifactId>
  <version>7.0.0</version>
</dependency>
```

### PR DESCRIPTION
**Pull request by mnriem**

### CODE REVIEW

File: maven-plugins/transformer-maven-plugin/README.md [hunk 1]
```diff
@@ -250,3 +252,17 @@ It is therefore necessary to configure the `maven-jar-plugin` as follows:
 
 The `help` goal displays help about the goals of the plugin.
 Call `mvn transformer:help -Ddetail=true -Dgoal=<goal-name>` to display configuration details for the specified goal.
+
+## Java 21 or later
+
+If you are using Java 21 or later you will need to configure the Maven plugin to use a specific version of the `biz.aQute.bnd` dependency.
```
Comment (bjhargrave) [inline maven-plugins/transformer-maven-plugin/README.md:258]:
```suggestion
If you are using Java 21 or later you will need to configure the Maven plugin to use a specific version of the `biz.aQute.bndlib` dependency.
```

File: maven-plugins/transformer-maven-plugin/README.md [hunk 2]
```diff
@@ -250,3 +252,17 @@ It is therefore necessary to configure the `maven-jar-plugin` as follows:
...
+
+E.g.
+
+```xml
+  <dependency>
+    <groupId>biz.aQute.bnd</groupId>
+    <artifactId>biz.aQute.bnd</artifactId>
```
Comment (bjhargrave) [inline maven-plugins/transformer-maven-plugin/README.md:265]:
```suggestion
    <artifactId>biz.aQute.bndlib</artifactId>
```

File: maven-plugins/transformer-maven-plugin/README.md [hunk 3]
```diff
@@ -250,3 +252,17 @@ It is therefore necessary to configure the `maven-jar-plugin` as follows:
 
 The `help` goal displays help about the goals of the plugin.
 Call `mvn transformer:help -Ddetail=true -Dgoal=<goal-name>` to display configuration details for the specified goal.
+
+## Java 21 or later
+
+If you are using Java 21 or later you will need to configure the Maven plugin to use a specific version of the `biz.aQute.bnd` dependency.
+
+E.g.
```
Comment (bjhargrave) [inline maven-plugins/transformer-maven-plugin/README.md:260]:
```suggestion
For example:
```

**Review by bjhargrave — CHANGES_REQUESTED** [review]
The correct artifactId is `bndlib`. See

https://github.com/eclipse/transformer/blob/50f259152cb5ae355074d676652834c6904790cb/maven-plugins/transformer-maven-plugin/pom.xml#L45-L48

**Comment by mnriem** [comment]
@bjhargrave Please review again. Changes have been applied.

**Review by bjhargrave — APPROVED** [review]

**Pull request merged**